lib/pure/algorithm.nim

```diff
@@ -732,11 +802,23 @@ proc rotatedLeft*[T](arg: openarray[T]; slice: HSlice[int, int], dist: int): seq
   ## Same as ``rotateLeft``, just with the difference that it does
   ## not modify the argument. It creates a new ``seq`` instead.
   ##
+  ## Elements outside of ``slice`` will be left unchanged.
+  ## If an invalid range (``HSlice``) is passed, it raises IndexError.
+  ##
+  ## ``slice``
+  ##   The indices of the element range that should be rotated.
+  ##
+  ## ``dist``
+  ##   The distance in amount of elements that the data should be rotated.
+  ##   Can be negative, can be any number.
+  ##
   ## **See also:**
   ## * `rotateLeft proc<#rotateLeft,openArray[T],HSlice[int,int],int>`_
   ## * `rotateLeft proc<#rotateLeft,openArray[T],int>`_
```

Comment:
Since there are two procs of the same name, and two of the similar name, I would add an explanation what each does to reduce confusion so the reader immediately knows which is which without the need to click first:

```
  ## **See also:**
  ## * `rotateLeft proc<#rotateLeft,openArray[T],HSlice[int,int],int>`_ for the in-place version of this proc
  ## * `rotatedLeft proc<#rotatedLeft,openArray[T],int>`_ for a version which rotates the whole container
```

(Notice that the second link is changed to `rotatedLeft`)

Reply:
Updated link description: 20067e844058b0bb96b415b5a1fe8b27c94019db